torch/csrc/jit/passes/quantization.cpp

```diff
@@ -116,9 +129,8 @@ static void addObserverFor(Value* v, Node* original_observer_node) {
   Node* observerNode =
       def->owningGraph()
           ->createClone(
-              &*original_observer_node, [&](Value* v) { return v; }, false)
-          ->insertAfter(def);
-
+              &*original_observer_node, [&](Value* v) { return v; }, false);
+  insertNodeHelper(observerNode, def, insert_info.second);
```

Comment:
I'm no expert in JIT IR, but how can you insert "before definition"? doesn't it violate order?